packages/sitecore-jss/src/personalize/cdp-service.ts

```diff
@@ -58,12 +62,20 @@ export class CdpService {
     const fetcher = this.config.dataFetcherResolver
       ? this.config.dataFetcherResolver<SegmentData>()
       : this.getDefaultFetcher<SegmentData>();
-    const response = await fetcher(endpoint, {
-      clientKey: this.config.clientKey,
-      browserId,
-      params,
-    });
-    return response.data;
+    try {
+      const response = await fetcher(endpoint, {
+        clientKey: this.config.clientKey,
+        browserId,
+        params,
+      });
+
+      return response.data;
+    } catch (error) {
```

Comment:
In this case, it means that any error which will be thrown will cause to return the default value, but we expected to handle only timeout